src/lucky_flow/find_element.cr

```diff
@@ -58,4 +61,12 @@ class LuckyFlow::FindElement
       end
     end
   end
+
+  private def all_elements  : Array(String)
+    session.find_elements(:css, "[flow-id]").map(&.attribute("flow-id")).uniq
+  end
+
+  private def nearest_match : String?
+    best_match = Levenshtein::Finder.find selector, all_elements, tolerance: 15
```

Comment:
This tolerance seems a bit high. I think adding a spec to make sure things that don't really look similar don't show a `did you mean` will help find a good tolerance.